tests/fsharpqa/testenv/src/ILComparer/ILComparer.fsproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
Note for future work, it might be a good idea to convert these tools to `netcoreapp2.1` (or 3.0).